src/components/PageHeader/_PageHeader.scss

```diff
@@ -79,6 +79,32 @@
   }
 }
 
+.coa-PageHeader--officialDocumentPage {
+  padding-top: $coa-spacing-large;
+
+  @include media($coa-medium-screen) {
```

Comment:
For consistency, let's use the mixins we've got for responsive styling `@mixin mobile`, etc.

Reply:
I dont remember where the comment you left about mixins was on Bob's PR, do you want me to change these to @mixin?

Reply:
Nope, don't change them